### PR TITLE
Dilpreet | Test | Invalid subscription id is getting pass in create company flow

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 1]
```diff
@@ -366,7 +366,7 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
         this.companyService.countryListBySubscriptionId(subscriptionId).pipe(takeUntil(this.destroyed$)).subscribe((response: any) => {
             if (response) {
                 this.countries = [];
-                Object.keys(response?.body).forEach(key => {
+                Object?.keys(response?.body).forEach(key => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/add-company/add-company.component.ts]: what's the reason of this change?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]
